patch/llpcPatchInOutImportExport.cpp

```diff
@@ -590,8 +591,8 @@ void PatchInOutImportExport::visitCallInst(
                 }
             case ShaderStageFragment:
                 {
-                    uint32_t interpMode = InterpModeSmooth;
-                    uint32_t interpLoc = InterpLocCenter;
+                    uint32_t interpMode = Builder::InOutInfo::InterpModeSmooth;
```

Comment:
Check if we could remove "Builder::". It is somewhat verbose.

Reply:
OK I will move InOutInfo out of Builder.